### PR TITLE
global: installable instance from template

### LINKED ISSUE #46: instance: remove `manage.py` script
Remove `manage.py` script from templates and rely only on installed `cli`.

--
Further reading http://click.pocoo.org/5/setuptools/

### PR DESCRIPTION
**Pull request by jirikuncar**
* Makes generated instance package installable.  (closes #46)

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

--
ping @lnielsen @tiborsimko

### CODE REVIEW

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```
Comment (lnielsen) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: I'm not sure it's a good idea to have the full blown setup.py. I think a a very minimal setup.py file is better in order to allow beginners to understand what's going on and what is the bare minimal need.

We can recommend in the docs that they can use our cookiecutter module if they want a full blown setup.

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    'pep257>=0.7.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'pytest>=2.8.0',
+]
+
+extras_require = {
+    'docs': [
+        "Sphinx>=1.3",
```
Comment (lnielsen) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: Single quotes

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+extras_require['all'] = []
+for reqs in extras_require.values():
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'Babel>=1.3',
+]
+
+install_requires = [
+    'Flask-BabelEx>=0.9.2',
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: we should probably add `invenio-base` here

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py (hunk 1 above)
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: Which part do you want to remove - Tests, Babel, Classifiers?

Comment (lnielsen) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: I would go with something like this:

```python
# -*- coding: utf-8 -*-

"""{{ cookiecutter.site_name }} setup script."""

from setuptools import find_packages, setup

setup(
    name='{{ cookiecutter.site_name }}',
    version="0.1.0.dev0",
    description=__doc__,
    packages=find_packages(),
    zip_safe=False,
    include_package_data=True,
    platforms='any',
    entry_points={
        'console_scripts': [
            '{{ cookiecutter.package_name }} = {{ cookiecutter.package_name }}.cli:cli'
        ],
    },
    install_requires=[
        'Flask-BabelEx>=0.9.2',
        'invenio-base>=1.0.0a3',
    ],
    setup_requires=[
        'Babel>=1.3',
    ],
)
```

Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: Feel free to amend my code then.

Comment (lnielsen) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: #47 now include an amended version of this PR.

**Pull request closed without merging**